Worktrolly/worktrolly
# feat(Bug: BoardPage): Show backlog when there is no sprint in a team

### Functionality:
Shows backlog when there is no sprint in the team

### Solution:
Removed some if conditions and used the changeSprint method to change the sprint to backlog

### Risk level:

### How to test:
ng serve
firebase emulators:start
Go to board and check a team with no sprint it
